api/v1alpha1/openapi.yaml

```diff
@@ -3031,6 +3031,56 @@ components:
               description: 'The directory in which the executable will be run from if it is left empty it will run from the users home directory.'
           required:
             - run
+    DeviceUpdatePolicySpec:
+      type: object
+      properties:
+        download:
```

Comment:
Should this be “downloadSchedule” or “downloadWindow”?

Otherwise, it’d seem that this field is subdividing “updatePolicy” by phases (download, update, rollback, …), meaning we’d put other policies related to “download” besides schedule (rate limits, retention policies, etc) here as well. Same with “update” and potential other phases (rollback?). IMO, that’d just increase nesting and be less extensible.